### PR TITLE
Color-App: first solution for part 2

### PR DESCRIPTION
**Pull request by e-Steffen**
If someone of group 1 (or several) would like to have a look aka make a code-review of my suggested solution, I would be thankful.

### CODE REVIEW

File: react/reacap-theme-creator-app/src/components/AllThemes.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+import { useState } from "react";
+import SingleTheme from "./SingleTheme.js";
+import ThemePreview from "./ThemePreview.js";
+import { IconChevronDown, IconChevronUp } from "@tabler/icons-react";
+
+export default function AllThemes({ theme }) {
+  const [detailView, checkDetailState] = useState(false);
+
+  function toggleView() {
+    checkDetailState(!detailView);
```
Comment (davide-esposito) [inline react/reacap-theme-creator-app/src/components/AllThemes.js]: maybe you could use "setDetailView" for better understanding

File: react/reacap-theme-creator-app/src/components/AllThemes.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
   return (
-    <>
-      {themes.map((theme) => {
-        return <Theme key={theme.id} theme={theme} />;
-      })}
-    </>
+    <article className="theme-overview">
+      <button className="view-toggle" onClick={toggleView}>
+        {detailView ? <IconChevronUp /> : <IconChevronDown />}
+      </button>
```
Comment (davide-esposito) [inline react/reacap-theme-creator-app/src/components/AllThemes.js]: you could also render your "<h2>{theme.name}</h2>" inside your button and delete it in your ThemePreview.js and SingleTheme.js.

**Review by davide-esposito — COMMENTED** [review]
Your code looks good to me, great job! Just added two comments, if you want you could change it, if not its also okay :)

File: react/reacap-theme-creator-app/src/components/AllThemes.js (hunk 2 above)
Comment (e-Steffen) [reply react/reacap-theme-creator-app/src/components/AllThemes.js]: thank you! I applied your suggestions for changes.

**Pull request merged**